src/Dscp/Crypto/Aeson.hs

```diff
@@ -0,0 +1,20 @@
+-- | Aeson instances for all types in Dscp.Crypto.
+
+module Dscp.Crypto.Aeson () where
+
+import Data.Aeson (FromJSON (..), ToJSON (..), object, withObject, (.:), (.=))
+
+import Dscp.Crypto.Encrypt (Encrypted (..))
+import Dscp.Crypto.Instances ()
+import Dscp.Util.Aeson (AsByteString (..), Base64)
+
+instance ToJSON a => ToJSON (Encrypted a) where
+    toJSON Encrypted{..} = object
+        [ "encrypted" .= eCiphertext
```

Comment:
Ciphertext is also going to be an arbitrary sequence of bytes, it also should be represented as `base64`, I assume? :thinking: 
Actually, I'd better provide a `Serialise` instance for `Encrypted` and store it in JSON in serialized way as `base64`. This way we'll be able to hide `Encrypted` constructor from exports, preventing it to be used unsafely.

Reply:
Makes sense.